syllabus/5-Substrate/9-Substrate-Interactions_Slides.md

```diff
@@ -54,6 +54,7 @@ almost all external communication happens over JSPN-RPC, so let's take a closer
 - Entirely transport agnostic.
 - Substrate based chains expose both `websocket` and `http` (or `wss` and `https`, if desired).
 
+ <!-- TODO(ank4n): Verify if this is correct. -->
```

Comment:
It's not correct. Both the http and ws servers are on port 9944 now and 9933 is no longer used.

Reply:
Thanks. Updated.